services/backend/src/router/user/user.js

```diff
@@ -1,16 +1,22 @@
 const { Router } = require("express");
 const { keycloak } = require("../../auth/keycloak");
 const user = require("../../core/user/user");
+const { UUIDValidator, createUserValidator } = require("./validator");
 
 const userRouter = Router();
 
 userRouter.get("/", keycloak.protect(), user.getUsers);
 
 userRouter
   .route("/:id")
-  .get(keycloak.protect(), user.getUserById)
-  .post(keycloak.protect(), user.createUser);
+  .get(keycloak.protect(), [UUIDValidator], user.getUserById)
+  .post(keycloak.protect(), createUserValidator, user.createUser);
```

Comment:
Should we check for UUID in the post ?

Reply:
the createUserValidator is an array including the UUIDValidator, should I leave it out of createUserValidator?

Reply:
I feel like we should separate them since UUID validation is done almost on all profiles and user related requests so we can write one method and share it for all of them. That way it would be easier to maintain.

Reply:
but createUserValidator is just an array, containing the UUIDValidator variable, not redeclaring the UUID body check